eks/multicam_smoother.py

```diff
@@ -2,31 +2,29 @@
 
 import numpy as np
 import pandas as pd
-from scipy.optimize import minimize
+import jax.numpy as jnp
 from sklearn.decomposition import PCA
 from typeguard import typechecked
 
 from eks.core import (
-    backward_pass,
-    compute_initial_guesses,
-    compute_nll,
-    forward_pass,
     jax_ensemble,
+    optimize_smooth_param, center_predictions
 )
 from eks.marker_array import (
     MarkerArray,
     input_dfs_to_markerArray,
     mA_to_stacked_array,
     stacked_array_to_mA,
 )
+#from eks.singlecam_smoother import singlecam_optimize_smooth
 from eks.stats import compute_mahalanobis, compute_pca
-from eks.utils import crop_frames, format_data, make_dlc_pandas_index
+from eks.utils import format_data, make_dlc_pandas_index
 
 
 @typechecked
 def fit_eks_mirrored_multicam(
     input_source: str | list,
-    save_file: str,
+    save_dir: str,
```

Comment:
should this stay "save_file"? below you use it this way:
```
os.makedirs(os.path.dirname(save_dir), exist_ok=True)
final_df.to_csv(f"{save_dir}")
```
indicating that this string should be a file, not a directory